<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="findLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Wyszukaj wizytę:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sortLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Sortuj wizytę:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueFindEditText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="listVisitsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>LISTA WIZYT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valueFindButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="valueFindComboboxButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
